docker/services/uwsgi/uwsgi.ini

```diff
@@ -0,0 +1,28 @@
+[uwsgi]
+uid = www-data
+gid = www-data
+master = true
+socket = 0.0.0.0:3031
+
+enable-threads = true
+processes = 30
+
+log-x-forwarded-for=true
+disable-logging = true
+; quit uwsgi if the python app fails to load
+need-app = true
+; when uwsgi gets a sighup, quit completely and let runit restart us
+exit-on-reload = true
```

Comment:
this file wasn't finished, it's a mix between LB and the current botbot interface